src/views/credentials/__tests__/useCredentialsQuery.test.ts

```diff
@@ -0,0 +1,46 @@
+import { type TableState } from '@mturley-latest/react-table-batteries';
+import { renderHook } from '@testing-library/react';
+jest.mock('../../../helpers/queryHelpers', () => ({
+  useServiceQuery: jest.fn(() => ({
+    data: { results: [], count: 0 },
+    isLoading: false,
+    isError: false,
+    error: null
+  }))
+}));
+import { useServiceQuery } from '../../../helpers/queryHelpers';
+import { type CredentialType } from '../../../types/types';
+import { useCredentialsQuery } from '../useCredentialsQuery';
+
+type CredentialsColumnKey = 'name' | 'type' | 'auth_type' | 'sources' | 'updated' | 'actions';
+type CredentialsSortableColumnKey = 'name' | 'type';
```

Comment:
These two are copy-pasted from [`useCredentialsQuery.ts`](https://github.com/quipucords/quipucords-ui/blob/main/src/views/credentials/useCredentialsQuery.ts#L14), because that module does not export them. I think it should. I can rework the module to export them, and then here I would import them instead of defining.

Reply:
Let's export them to avoid duplicates in tests

Action item: let's check with cory what should be the best approach here, concentrating all in one file (like type per example) or making it a bit more modular.

Reply:
Export is done in 5241404ead142d211f8d54f55ef89420b021a6ab 

I created DISCOVERY-1048 to track "use centralized or modular, but be consistent".